pkg/core/manifest.go

```diff
@@ -21,19 +21,84 @@ import (
 	"fmt"
 	"io/ioutil"
 
-	"gopkg.in/yaml.v2"
+	"github.com/ghodss/yaml"
 )
 
-const manifestVersion_0_1 = "0.1"
+const ManifestVersion = "0.1"
```

Comment:
@ericbottard chose the name earlier today. This poses some questions:
1. Does the constant now need to be exported?
2. Should the constant name include the version?

I guess we can claim YAGNI for question 2 and add in the version if and only if we end up supporting multiple manifest versions in a single CLI.

Reply:
I can change the const name back.

We don't need to know the version in order to unmarshal since unknown values are ignored. Since there's an affirmative check for a valid version, we are covered. How we handle multiple manifest versions is unknown, but we can cross that bridge when we get to it.